cmd/otel-allocator/README.md

```diff
@@ -0,0 +1,82 @@
+# Target Allocator
+
+The TargetAllocator is an optional separately deployed component of an OpenTelemetry Collector setup, which is used to
+distribute targets of the PrometheusReceiver on all deployed Collector instances.
+
+# Design
+
+If the Allocator is activated, all Prometheus configurations will be transferred in a separate ConfigMap which get in
```

Comment:
> all Prometheus configurations

Is it OTEL p8s receiver configuration or as well p8s CRs?

Reply:
This applies to the configuration of the `prometheus` receiver in the collector configuration.  The Prometheus operator CRs are observed separately after the target allocator has been started, at which point it further modifies the service discovery configuration it is using.